omnigibson/objects/dataset_object.py

```diff
@@ -104,6 +104,10 @@ def __init__(
         load_config = dict() if load_config is None else load_config
         load_config["bounding_box"] = bounding_box
 
+        # TODO: Remove once meshes are fixed
+        from omnigibson.utils.bddl_utils import DO_NOT_REMESH_CLOTHS
+        load_config["remesh"] = model not in DO_NOT_REMESH_CLOTHS.get(category, set())
```

Comment:
change to BAD_CLOTH_MODELS and immediately raise an error if self.prim_type == PrimType.CLOTH and self.model in BAD_CLOTH_MODELS

Reply:
done